build.gradle.kts

```diff
@@ -184,3 +185,30 @@ project.afterEvaluate {
         }
     }
 }
+
+tasks.register("publishVersion") {
```

Comment:
What is the task actually doing? Could it be that the task logic is missing?

Reply:
It triggers the MavenPublication "bootJava" (see usage in workflows/publish.yml). But yeah it's not really obvious that the "dependsOn" trigger that one